Resources/views/FragmentAdmin/edit_one_association_script_fragment.html.twig

```diff
@@ -0,0 +1,51 @@
+{% autoescape false %}
```

Comment:
Sorry miss that one. Converted into data attributes and editOneAssociationFragment.js script